GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MountTests.cs

```diff
@@ -210,6 +210,21 @@ public void MountRegeneratesAlternatesFileWhenMissingFromDisk()
             alternatesFilePath.ShouldBeAFile(this.fileSystem).WithContents(objectsRoot);
         }
 
+        [TestCase]
+        public void MountCanProcessSavedBackgroundQueueTasks()
+        {
+            this.Enlistment.UnmountGVFS();
+
+            string deleteFilePath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4", "ReadDeepProjectedFile.cpp");

```

Comment:
Could you also validate the `OnFolderRenamed` (10) background task in this test?

That was the one other scenario that @kewillford called out in the comment on `FileSystemCallbacks.cs`.

Reply:
As discussed offline we are going to punt that test for now